src/Utilities/hooks/api/utils/normalize.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+import normalize from 'json-api-normalizer';
+
+export const defaultIntialState = {
+    loading: false,
+    error: null
+};
+
+export const initialStateFor = function (reducerName, initialObject = []) {
+    let initState = Object.assign({}, defaultIntialState);
+    initState[reducerName] = initialObject;
+    return initState;
+};
+
+export const successMessage = (base) => {
+    return base + '_FULFILLED';
+};
+
+export const pendingMessage = (base) => {
+    return base + '_PENDING';
+};
+
+export const failureMessage = (base) => {
+    return base + '_REJECTED';
+};
+
+export const findItemById = (id, array) =>
+    array.find((item) => parseInt(item.id) === parseInt(id));
```

Comment:
`id` and `item.id` here are both UUID strings

```suggestion
    array.find((item) => item.id === id);
```